### ISSUE TITLE
Add Host OS test case for cpupower modification of governor settings

### ISSUE THREAD

**Issue by rquander** [open-power-host-os/tests#25]
Add an automated test to the Host OS Test Suite to test cpupower modification of governor settings.

**Comment by rquander** [comment]
Test case enhancement is still in progress.

**Comment by rquander** [comment]
@PavithraDP submitted a new PR for this test in the avocado-misc-tests repository ([#228](https://github.com/avocado-framework-tests/avocado-misc-tests/pull/228)).  Once all review comments have been resolved and the PR is merged, I'll close this Issue.

**Comment by rquander** [comment]
The new test in [avocado-misc-tests PR #228](https://github.com/avocado-framework-tests/avocado-misc-tests/pull/228) is merged.  I need to add this test to the Host test sanity.cfg file and then close this Issue.

**Comment by rquander** [comment]
Resolved with Pull Request #50
